docs/modeling-your-data/modeling-your-data-with-dbt/dbt-advanced-usage/dbt-advanced-operation/index.md

```diff
@@ -41,6 +42,42 @@ dbt run --select +snowplow_mobile.screen_views --vars "{'models_to_run': '$(dbt
 </TabItem>
 </Tabs>
 
+## Custom re-running of specific models
+
+There may be times where you need to re-run some or all of the models that for a period of data, potentially due to incorrect supplemental data brought into a model, or an error that was made in your model code itself. There are two methods to do this; one is simpler but must complete in a single run, the second is more complex but be processed over multiple runs. Either will work for any custom models assuming they were built using the [snowplow incremental materialization](/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-advanced-usage/dbt-incremental-materialization/index.md).
```

Comment:
```suggestion
There may be times where you need to re-run some or all of the models for a period of data, potentially due to incorrect supplemental data brought into a model, or in case of a mistake in your model code itself. There are two methods to do this: one is simpler but must complete in a single run, the other is more complex but can be executed over multiple runs. Either will work for any custom models, assuming they were built using [Snowplow incremental materialization](/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-advanced-usage/dbt-incremental-materialization/index.md).
```